.standard_todo.yml

```diff
@@ -2,9 +2,8 @@
 # Remove from this list as you refactor files.
 ---
 ignore:
-- bundle/bin/*
-- db/schema.rb
-- db/data_schema.rb
-- lib/bangladesh/import/to_patients.rb
-- app/controllers/admin/facilities_controller.rb
-- app/controllers/admins_controller.rb
+  - bundle/bin/*
```

Comment:
This is proper yml formatting, [as seen here](https://github.com/testdouble/standard/blob/master/.standard.yml).